package.json

```diff
@@ -26,7 +26,9 @@
     "async": "2.1.2",
     "cloudant-nano": "6.5.0",
     "debug": "2.2.0",
-    "request": "^2.81.0"
+    "lockfile": "1.0.3",
```

Comment:
Any reason not to put a range on this to try and keep it up to date? Seems like the sort of thing where a bug fix or optimizations/tweaks/fixes for future Node.js versions could be handy.